src/utils/fee.ts

```diff
@@ -15,16 +33,16 @@ import { formattedNum } from 'utils/index'
 export function getFormattedFeeAmountUsd(trade: Aggregator, feeConfig: FeeConfig | undefined) {
   if (feeConfig) {
     const amountInUsd = new Fraction(
-      parseUnits(trade.amountInUsd.toString(), RESERVE_USD_DECIMALS).toString(),
-      JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(RESERVE_USD_DECIMALS)),
-    )
-    // feeAmount might < 1.
-    const feeAmountFraction = new Fraction(
-      parseUnits(feeConfig.feeAmount, RESERVE_USD_DECIMALS).toString(),
+      parseUnits(toFixed(trade.amountInUsd), RESERVE_USD_DECIMALS).toString(),
       JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(RESERVE_USD_DECIMALS)),
     )
-    const feeAmountDecimal = feeAmountFraction.divide(BIPS_BASE)
     if (amountInUsd) {
+      // feeAmount might < 1.
+      const feeAmountFraction = new Fraction(
+        parseUnits(feeConfig.feeAmount, RESERVE_USD_DECIMALS).toString(),
```

Comment:
do we need toFixed for `feeConfig.feeAmount`

Reply:
no, it already string